samples/aspnetcore-authz-with-ext-authz-service/Contoso.API/Authorization/CanGetWeatherAuthorizationHandler.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.AspNetCore.Authorization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Contoso.API.Authorization
```

Comment:
Use [File Scoped Namespaces](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces)

Reply:
updated, I did not know about this feature, appreciate the link!